src/export/bops/index.ts

```diff
@@ -159,6 +158,29 @@ export function formatProposalDetails({
     );
     if (!isTypeForBopsPayload(crumb.type) || !validKey) continue;
 
+    const metadata = (() => {
+      const metadata: QuestionMetaData = {};
+      if (hasSections) {
+        metadata.section_name = currentSectionName;
+      }
+      if (crumb.autoAnswered) metadata.auto_answered = true;
+      if (node.data?.policyRef) {
+        const policyRefs = parsePolicyRefs(node.data.policyRef as string);
+        if (policyRefs.length) metadata.policy_refs = policyRefs;
+      }
+      return metadata;
+    })();
```

Comment:
Untouched block, just moved higher in the function so that we can pass `metadata` along.